install-scripts/install_ipfix-rita.sh

```diff
@@ -0,0 +1,541 @@
+	while [ -n "$potential_target" ] && ! can_ssh "$potential_target" ; do
+		echo "What is the hostname or IP address where you would like to send IPFIX logs to?  Enter 127.0.0.1 if you want to run on this system.  Press Ctrl-C to exit." >&2
+		read potential_target <&2
+		if [ -n "$potential_target" -a "$potential_target" != '127.0.0.1' ]; then
+			echo "Do you access that system with a username different than ${USER}?  If so, enter that username now, otherwise just press enter if your account is ${USER} on the remote system too." >&2
+			read potential_user <&2
+			if [ -n "$potential_user" ]; then
+				potential_target="${potential_user}@${potential_target}"
+			fi
+		fi
+	done
+
+	echo "$potential_target"
+} #End of validated_ssh_tartget
+
+get_line() {
+	rita_file=$1
+	search_key=$2
+
+	#get the line that matches the search key that isn't commented out
+	line=`cat $rita_file | grep $search_key`
+	line=`echo $line | cut -d ' ' -f 2`
+
+	echo $line
+} #End of get line
+
+get_rita_data_interactive() {
+	IPFIX_RITA_NETWORK_GATEWAY=$(docker inspect ipfix_rita_default --format "{{with (index .IPAM.Config 0)}}{{.Gateway}}{{end}}")
+	RITA_MONGO_URI="mongodb://$IPFIX_RITA_NETWORK_GATEWAY:27017"
+
+	echo "" >&2
+	echo "IPFIX-RITA needs to write to a MongoDB database controlled by RITA." >&2
+	echo "By default, this installer assumes RITA and MongoDB are installed on the Docker host." >&2
+	echo "In order to support this type of installation, you will need to" >&2
+	echo "add the suggested Docker interface below to the list of bindIP's in /etc/mongod.conf." >&2
+	echo "If needed, please do so, and restart MongoDB before continuing." >&2
+	echo "Restart MongoDB with \$ sudo systemctl restart mongod.service" >&2
+	echo "Note: the default configuration is not recommended. IPFIX-RITA will likely perform" >&2
+	echo "better if it is installed on a machine separate from RITA/ MongoDB." >&2
+	echo "" >&2
+
+	echo "What MongoDB URI should IPFIX-RITA use to contact the RITA database [$RITA_MONGO_URI]: " >&2
+	read RITA_MONGO_URI <&2
+	if [  -n "$RITA_MONGO_URI" -a  "$RITA_MONGO_URI" != '127.0.0.1' ]; then
+		rita_uri=`strip_mongo $RITA_MONGO_URI`
+		RITA_MONGO_URI=`add_mongo $RITA_MONGO_URI`
+	        echo -n "Do you access that system with a username different than ${USER} (Y/N)" >&2
+	        if askYN ; then
+	                echo "Enter that username now, otherwise just press enter if your account is ${USER} on the remote system too." >&2
+	                read rita_user
+	                if [ -n "$rita_user" ]; then
+	                        rita_system="${rita_user}@${rita_uri}"
+	                fi
+	        fi
+					#This will check if the Rita system is accessible to our installer
+					#TODO, this cases issues, we have mongodb://127.0.0.1:27017 then we
+					# have ritauser@mongodb://127.0.0.1:27017 need to detect if they have
+					# mongodb and the port before trying to validate
+	        rita_system=`validated_ssh_target rita "$rita_system"`
```

Comment:
We don't need to access the rita system with ssh in order to install ipfix-rita. (Unless we plan on installing rita as well)